### DISCUSSION TITLE
QLog configuration

### DISCUSSION THREAD

**Discussion by poisonriver** [netty/netty-incubator-codec-quic#727] [Ideas]
I found that it's not very convenient to have a QLog configured for the whole server. I'm thinking that adding IpFilterRule set to the QLogConfiguration will make it more flexible in terms of debugging/analyzing issues.

**Comment by normanmaurer** [comment]
So you would filter out what we receive via `quiche` ?

**Reply by poisonriver** [reply]
> So you would filter out what we receive via `quiche` ?

I will check the source address against the qlog config before enabling qlog for a connection

**Reply by normanmaurer** [reply]
But this would also mean that you might loose logs if a connection migration happens

**Reply by poisonriver** [reply]
> But this would also mean that you might loose logs if a connection migration happens

fair enough, it's not a magic bullet. but enabling qlog for every connection is a bit overhead sometimes

**Reply by normanmaurer** [reply]
I guess as long as its "optimal" and you can enable it for everything via some wildcard this is ok :)

**Reply by poisonriver** [reply]
so I'm creating a PR?

**Reply by normanmaurer** [reply]
yes please
